include/merlin_hashtable.cuh

```diff
@@ -308,22 +314,13 @@ class HashTable {
       }
 
       {
-        static_assert(sizeof(value_type*) == sizeof(uint64_t),
-                      "[HierarchicalKV] illegal conversation. value_type pointer "
-                      "should be 64 bit!");
-
         const size_t N = n;
-        thrust::device_ptr<uint64_t> d_dst_ptr(
-            reinterpret_cast<uint64_t*>(d_dst));
+        thrust::device_ptr<uintptr_t> d_dst_ptr(
```

Comment:
Pls add the header of <cstdint>for the uintptr_t.

Reply:
Done!